.gitignore

```diff
@@ -12,3 +12,5 @@ assets/js/*.deps.json
 assets/js/*.map
 built
 /amphtml
+.env
```

Comment:
Should we add a `.env.example` file with `GOOGLE_FONTS_API_KEY=` as the content? (yes I would rename it). Then we can document it in `contributing.md`

Reply:
Done